operator/redisfailover/service/generator.go

```diff
@@ -281,6 +281,196 @@ func generateHAProxyService(rf *redisfailoverv1.RedisFailover, labels map[string
 	}
 }
 
+func generateHAProxyRedisSlaveDeployment(rf *redisfailoverv1.RedisFailover, labels map[string]string, ownerRefs []metav1.OwnerReference) *appsv1.Deployment {
```

Comment:
Had we considered renaming the existing `generateHAProxyDeployment`, etc to `generateHAProxyRedisMasterDeployment`, etc for consistency / clarity?

Reply:
renamed